src/chi/components/card/card.scss

```diff
@@ -124,7 +124,54 @@
       &.-noBorder {
         box-shadow: none;
       }
+    }
+
+    &.-empty {
+      background-color: transparent;
+      border: dashed rgba(set-color(black), 0.12);
+      transition: all 0.2s;
+
+      & .a-card__content {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin: 1rem;
+
+        & .a-icon {
+          color: set-color(grey, 60);
+          margin-bottom: 1rem;
+        }
+
+        & span {
+          font-size: $text-large;
+          font-weight: 600;
+        }
+      }
+
+      &:hover {
+        border: dashed $brand-color;
+        color: $brand-color;
+        cursor: pointer;
+
+        & .a-icon {
+          color: $brand-color;
+        }
+      }
+
+      &:active,
```

Comment:
@mattnickles do we need an :active state for the empty cards? @davidojedacl, if it is not necessary, remove this part

Reply:
@jllr @davidojedacl active as in an active/selected state, not a pseudo class. no pseudo class is needed here.

Example:
This is a card with an -active modifier that would render very similarly to an active picker component. We're simply changing the border-color to $brand and the box-shadow to a 1px outset colored $brand which will visually appear as a 2px border.
![image](https://user-images.githubusercontent.com/130654/50245715-0ec8ff80-0388-11e9-83f0-a2cfc57b4509.png)